### PR TITLE
add menu-key-event to 'Menu widget'

### PR DESCRIPTION
**Pull request by Commodore64user**
As seen here https://github.com/koreader/koreader/pull/11918#issuecomment-2137290809

This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/11966)

### CODE REVIEW

File: frontend/ui/widget/menu.lua [hunk 1]
```diff
@@ -963,6 +963,7 @@ function Menu:init()
     if Device:hasKeys() then
         -- set up keyboard events
         self.key_events.Close = { { Input.group.Back } }
+        self.key_events.LeftButtonTap = { { "Menu" } }
```
Comment (Frenzie) [inline frontend/ui/widget/menu.lua:966]: `LeftButtonTap` becomes a bad name if you add this.

Comment (Commodore64user) [reply frontend/ui/widget/menu.lua:966]: What do you mean by "becomes a bad name"? That is what the existing function is called

Comment (Frenzie) [reply frontend/ui/widget/menu.lua:966]: It's changing the original context in which it was named, making it confusing. Something like `WidgetMenuToggle`? Pinging @hius07 who wrote it for some thoughts.

Comment (hius07) [reply frontend/ui/widget/menu.lua:966]: If it works, I think it's okay.

**Pull request merged**